code/mob/living/critter/spider.dm

```diff
@@ -27,6 +27,7 @@
 	health_burn = 50
 	health_burn_vuln = 0.65
 	reagent_capacity = 100
+	void_mindswappable = TRUE
```

Comment:
Nah spiders are annoying with mind-swaps, mostly because they spew out a lot of little NPC spiders everywhere - see also other comments that this is why mindswaps were restricted in the first place.
```suggestion
```